.github/workflows/edge.yml

```diff
@@ -53,5 +53,6 @@ jobs:
           tag_name: edge
           body: Automated build from commit ${{ github.sha }}
           files: |
+            PLUG.md
             silversearch.plug.js
-            PLUG.md
\ No newline at end of file
+            silversearch-tokenizer-chinese.wasm
```

Comment:
I'm not a fan of shipping multiple files for Silversearch. I'm also not a fan that this file is ~3.something MB. I'd be most interested if this could be solved similarly to Omnisearch or the pdf plug. I.e. a separate plug, which contains all the complex code, in this case the wasm stuff, which is shipped separately and user can install it and then Silversearch communicates with that plug to do tokenization. Maybe even make it a general api, so people could build tokenizers for other languages too.